internal/log/log.go

```diff
@@ -3,20 +3,56 @@
 package log
 
 import (
-	"github.com/sirupsen/logrus"
+	"bytes"
+	"fmt"
+
+	"github.com/go-logr/logr"
 )
 
-var l *logrus.Logger
+const (
+	DBG int = 1
+	TRC int = 2
```

Comment:
Are these numbers just random? Looking at the implementation it doesn't seem like it's a typical linear scale ie 1-5 (trace, debug, warn, info, error) for the levels. And seems that it can be any number.